arm/Microsoft.KubernetesConfiguration/extensions/readme.md

```diff
@@ -45,7 +45,6 @@ For Details see [Prerequisites](https://docs.microsoft.com/en-us/azure/azure-arc
 **Optional parameters**
 | Parameter Name | Type | Default Value | Description |
 | :-- | :-- | :-- | :-- |
-| `autoUpgradeMinorVersion` | bool | `True` | Flag to note if this extension participates in auto upgrade of minor version, or not. |
```

Comment:
I'd suggest to add a comment to the `version` parameter indicating what happenes if you don't provide a version

Reply:
Done